### ISSUE TITLE
It is hard to see the icon of the date in the 'Shared Access Signature' dialog under the 'High Contrast - Black' theme

### ISSUE THREAD

**Issue by v-limu** [microsoft/AzureStorageExplorer#4173]
**Storage Explorer Version**: 1.18.1
**Build Number**: 20210306.4
**Branch**: main
**Platform/OS:** Windows 10/ Linux ubuntu 16.04/ MacOS Catalina
**Architecture**: ia32/x64
**Regression From**: Previous release(1.18.1)

## Steps to Reproduce ##

1. Open Storage Explorer -> Go to the 'View' menu -> Change the theme to 'High Contrast - Black'.
2. Select one storage account -> Click 'Get Shared Access Signature...'.
3. Observe the 'Shared Access Signature' dialog -> Check whether all icons are clear. 

## Expected Experience ##
All icons are clear. 
![image](https://user-images.githubusercontent.com/54055206/110286031-2d764800-801f-11eb-9e9c-b1fbb64f497f.png)

## Actual Experience ##
It is hard to see the icon of the date.
![image](https://user-images.githubusercontent.com/54055206/110285770-b5a81d80-801e-11eb-8f42-09c286a0cb32.png)

## Additional Context ##

1. This issue also reproduces in the 'Access Policies' dialog.
2. This issue also reproduces in the 'Undelete by Date...' dialog.

**Comment by MRayermannMSFT** [comment]
Looked into this. Appears to be a chrome change. We could do something like this to change the color, but it is pretty hacky: https://stackoverflow.com/questions/62162645/change-color-of-chromes-calendar-icon-in-html-date-input

I'm going to open a chromium bug.

**Comment by MRayermannMSFT** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=1188389

**Comment by v-xianya** [comment]
Verified this issue on the build main/20220615.4. Fixed. so closed.
![image](https://user-images.githubusercontent.com/41351993/173752073-78254f74-968e-48d1-97ac-40d0e0a6fd8b.png)
